src/main/java/se/citerus/dddsample/domain/shared/DomainEntity.java

```diff
@@ -4,7 +4,7 @@
  * An entity, as explained in the DDD book.
  *  
  */
-public interface Entity<T> {
+public interface DomainEntity<T> {
```

Comment:
Why was this class renamed?

Reply:
yes,  IMHO, this class should not use a single word.